src/screens/ProfileScreen/index.js

```diff
@@ -53,7 +54,6 @@ import {Analytics} from '../../libraries/analytics/firebaseAnalytics';
 import {ButtonNewPost} from '../../components/Button';
 import {Context} from '../../context';
 import {DEFAULT_PROFILE_PIC_PATH, SOURCE_MY_PROFILE} from '../../utils/constants';
-import {KarmaScore} from './elements/KarmaScore';
 import {
   changeRealName,
   getMyProfile,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `KarmaScore` is removed. Ensure that this component is not used anywhere else in the file. If it is, this could cause a reference error.
```diff
+ import {KarmaScore} from './elements/KarmaScore';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->